src/renderer/views/Playlist/Playlist.js

```diff
@@ -418,6 +423,20 @@ export default defineComponent({
       showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist does not exist'))
     },
 
+    checkPlaylist: function () {
```

Comment:
I can't think of a better one yet but the method is to show notice (not warning) and it shows sometimes only
Let me know if you can think of a better name

```suggestion
    showNoticesSometimes: function () {
```